maestro/env-sample.js

```diff
@@ -0,0 +1,6 @@
+// Define test specific accounts here.
+// eslint-disable-next-line no-undef
+output.logins = {
+  C314: { USERNAME: '', PASSWORD: '', PIN: '', BACKUP_CODE: '' },
+  C753: { USERNAME: '', PASSWORD: '', PIN: '', BACKUP_CODE: '' }
```

Comment:
I don't know how your sidebar with RJ went. These names could be hard to work with. I could see using names like "noPinUser", "no2faUser", "eosUser", "unactivatedEosUser", etc. being easier long-term, but like I said, this is really up to RJ about how he wants the workflow to go.

Reply:
As discussed, the test file name containing the test description (eg `C311-relaunch-no-crash.yaml`) means we do not need to add further verbiage to the property names of the `output.logins.<name>...` for credentials to maintain readability. In either case, the scope of this PR has been reduced to just the first two tests (`C310-launch-no-crash.yaml` & `C311-relaunch-no-crash.yaml`).